### ISSUE TITLE
Hypoplastic optic chiasm

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7618]
**Preferred term label:**
Hypoplastic optic chiasm

**Synonyms**
Small optic chiasm

**Definition (free text, please give PubMed ID)**
Developmental defect characterized by undergrowth of the optic chiasm.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 615113 
2/3
PMID: 23312594
also PMID: 23591992 1/3

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
